swupd/manifest.go

```diff
@@ -531,3 +531,38 @@ func getManifestVerFromMoM(mom *Manifest, b *Manifest) uint32 {
 
 	return 0
 }
+
+// maxFullFromManifest maximizes all file entries in mf that are also in bundle
+func maxFullFromManifest(mf *Manifest, bundle *Manifest) {
+	mf.sortFilesName()
```

Comment:
Full manifest is being sorted multiple for each manifest.

One sort in maximizeFull should be enough. And document this expectation (or inline this function).